app/grants/clr.py

```diff
@@ -67,6 +67,10 @@ def generate_grant_pair(grant):
             else:
                 unique_contributions[profile] = amount
 
+    if len(unique_contributions) == 1:
+        profile = next(iter(unique_contributions))
+        unique_contributions['_' + profile] = 1
+
```

Comment:
🤔

Reply:
I tried cleaning up using a rebase and used the wrong commit. I'm working on it and should have it fixed here in a moment.